partcad/src/partcad/context.py

```
@@ -169,9 +169,10 @@ def import_project(self, parent, project_import_config):
                 self._projects_being_loaded[name] = True
 
                 # Depending on the project type, use different factories
-                if not "type" in project_import_config or project_import_config["type"] == "local":
+                if "type" not in project_import_config or project_import_config["type"] == "local":
                     with pc_logging.Action("Local", name):
                         rfl.ProjectFactoryLocal(self, parent, project_import_config)
+                        pc_logging.debug("Local project loaded: %s" % name)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)